widget/playground/src/utils/configs.ts

```diff
@@ -17,6 +17,10 @@ interface Configs {
  */
 export const RANGO_PUBLIC_API_KEY = 'c6381a79-2817-4602-83bf-6a641a409e32';
 const WC_PROJECT_ID = 'e24844c5deb5193c1c14840a7af6a40b';
+export const TREZOR_MANIFEST = {
+  appUrl: 'https://rango.exchange/',
+  email: 'hi@rango.exchange',
```

Comment:
update these two email & urls to unique values for widget as well (`hi+...@rango....`).

Reply:
fixed